executor/executor.go

```diff
@@ -151,9 +151,9 @@ func doRunCommand(guid, path string, emitter screwdriver.Emitter, f *os.File, fR
 }
 
 // Executes teardown commands
-func doRunTeardownCommand(cmd screwdriver.CommandDef, emitter screwdriver.Emitter, path, shellBin, exportFile, sourceDir string) (int, error) {
+func doRunTeardownCommand(cmd screwdriver.CommandDef, emitter screwdriver.Emitter, path, shellBin, exportFile, sourceDir string, aggExitCode int) (int, error) {
 	shargs := []string{"-e", "-c"}
-	cmdStr := "export PATH=$PATH:/opt/sd && " +
+	cmdStr := "export PATH=$PATH:/opt/sd AGG_EXIT_CODE=" + strconv.Itoa(aggExitCode) + " && " +
```

Comment:
can we call this `SD_STEP_EXIT_CODE` and document that any non-zero value indicates build failure. 

can this be translated to build status ?

Reply:
yes, i can rename it.
no, should leave it up to each teardown to interpret it differently, and teardown step itself can fail